backend/elasticsearch-aws/pom.xml

```diff
@@ -12,7 +12,7 @@
         <groupId>org.hibernate.search</groupId>
         <artifactId>hibernate-search-parent-public</artifactId>
         <version>6.2.0-SNAPSHOT</version>
-        <relativePath>../../parents/public</relativePath>
+        <relativePath>../../build/parents/public</relativePath>
```

Comment:
I've noticed that some of these relative paths were pointing to a folder (like this one) while others (e.g. `        <relativePath>../../../build/parents/integrationtest/pom.xml</relativePath>`) are referring to a pom. Maybe it's worth making these more consistent?

Reply:
Right. I added a commit to do just that. Let's hope that consistency is there to stay... :)